<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fachabteilungen</class>
 <widget class="QDialog" name="Fachabteilungen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>797</width>
    <height>703</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fachabteilungen</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>260</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>191</height>
    </size>
   </property>
   <property name="title">
    <string>Fachabteilungen</string>
   </property>
   <widget class="QLabel" name="lblabteilungsleiter">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>45</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Abteilungsleiter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblfachabteilungabk">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fachabteilung:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblgebeaude">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>95</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gebäude:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblfachabteilungbez">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>122</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezeichnung:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFachabteilungabk">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStock">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>144</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFachabteilungbez">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>121</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAbteilungsleiter">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>44</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblstock">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>145</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stock:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGebaeude">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>94</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btCSV">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>195</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Report</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</normaloff>../../_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btAbbrechen">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>195</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>115</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hauptmenü</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btExcel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>195</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>115</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Excel</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btSuchen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>195</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Suchen</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btLoeschen">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>195</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Löschen</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</normaloff>../../_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btClear">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>195</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Leeren</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/-clear_90704.ico</normaloff>../../_Symbole/UsedSymbols/-clear_90704.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btSpeichern">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>195</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/save_3621.ico</normaloff>../../_Symbole/UsedSymbols/save_3621.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btHilfe">
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>195</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hilfe</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../_Symbole/UsedSymbols/info_13213.ico</normaloff>../../_Symbole/UsedSymbols/info_13213.ico</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_INFO">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>683</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
